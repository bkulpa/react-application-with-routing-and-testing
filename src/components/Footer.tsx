import { FC } from 'react';
const currentYear: number = new Date().getFullYear();

export const Footer: FC = () => {
    return (
        <p>Copyright Â© {currentYear} by Bartosz Kulpa </p>
    );
};
