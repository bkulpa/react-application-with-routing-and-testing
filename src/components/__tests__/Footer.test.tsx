import { render } from '@testing-library/react';
import { Footer } from '../Footer';

describe('Footer', () => {
    it('should render', () => {
        // before
        const { asFragment } = render(<Footer />);
        // result
        expect(asFragment()).toMatchSnapshot();
    });
});