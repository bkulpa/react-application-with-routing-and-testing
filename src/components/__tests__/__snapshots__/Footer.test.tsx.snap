// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render 1`] = `
<DocumentFragment>
  <p>
    Copyright Â© 2022 by Bartosz Kulpa 
  </p>
</DocumentFragment>
`;
